src/imitation/policies/interactive.py

```diff
@@ -1,36 +1,58 @@
+"""Interactive policies that query the user for actions."""
+
 import abc
-from typing import Optional, List
+import collections
+import typing
```

Comment:
Our style guide allows importing types directly from typing (i.e. `from typing import Optional` is permissible) although it's not obligatory -- fine to use this style if you prefer. https://google.github.io/styleguide/pyguide.html#2241-exemptions

Reply:
Thanks, good to know!